Not running okbuck for mudoles that contains -h in the name

Hi folks!

I can see that, if I try to run many build at once using buckw, it doesn't run okbuck as is expected. 

If I run: 

```
./buckw build \
    //android/android-foundation/fondation:src_debug \
    //android/android-bonafont/bonafont:src_debug \
```

It goest just fine. But if I add a third library, it doesn't run okbuck and doesn't generate de BUCK files. 

Likes this:

```
./buckw build \
    //android/android-foundation/fondation:src_debug \
    //android/android-bonafont/bonafont:src_debug \
    //android/android-help/help:src_debug \
```

I end up with: 

```
BUILD FAILED: //android/android-foundation/fondation:src_debug references non-existent directory android/android-foundation/fondation
```

The error is expected, since okbuck didn't run

--- a/buckw
+++ b/buckw
@@ -236,7 +236,7 @@ setupBuckRun ( ) {
 # Handle parameters and flags
 handleParams ( ) {
    # Go directly to the kill command, help command, or --help option. Do not run okbuck.
-   if [[ "kill" == $1 || "help" == $1 || $@ == *"--help"* || $@ == *"-h"* ]]; then
+   if [[ "kill" == $1 || "help" == $1 || $@ == *"--help"* || $@ == *" -h "* ]]; then
       SKIP_OKBUCK=true
    fi
 }

--- a/buildSrc/src/main/rocker/com/uber/okbuck/template/config/BuckWrapper.rocker.raw
+++ b/buildSrc/src/main/rocker/com/uber/okbuck/template/config/BuckWrapper.rocker.raw
@@ -231,7 +231,7 @@ setupBuckRun ( ) {
 # Handle parameters and flags
 handleParams ( ) {
    # Go directly to the kill command, help command, or --help option. Do not run okbuck.
-   if [[ "kill" == $1 || "help" == $1 || $@@ == *"--help"* || $@@ == *"-h"* ]]; then
+   if [[ "kill" == $1 || "help" == $1 || $@@ == *"--help"* || $@@ == *" -h "* ]]; then
       SKIP_OKBUCK=true
    fi
 }

